monkestation/code/modules/new_antagonists/slasher/abilities/recall_machette.dm

```diff
@@ -0,0 +1,88 @@
+/datum/action/cooldown/slasher/summon_machette
+	name = "Summon Machette"
+	desc = "Summon your machete to your active hand, or create one if it doesn't exist. This machete deals 15 BRUTE on hit, and stuns on throw."
+
+	button_icon_state = "summon_machete"
+
+	cooldown_time = 15 SECONDS
+
+	var/obj/item/slasher_machette/stored_machette
```

Comment:
might be worth changing this to a weakref if you have time

Reply:
machettes are tagged with indestructible so they shouldn't be destroyed hence a weakref isn't needed

Reply:
ah yeah, I mean technically they could from something like the SM but I think thats rare enough it should be fine